web-components/src/components/tabs/TabPanel.tsx

```diff
@@ -19,7 +19,7 @@ const TabPanel: React.FC<TabPanelProps> = props => {
       aria-labelledby={`tab-${index}`}
       {...other}
     >
-      <Box sx={{ p: 3 }}>{children}</Box>
+      <Box sx={{ p: 3 }}>{hidden ? null : children}</Box>
```

Comment:
Before inactive tabs were still rendering and potentially triggering requests because they were just hidden.